microsite/data/plugins/scaleops.yaml

```diff
@@ -0,0 +1,9 @@
+title: ScaleOps
+author: terasky.com
+authorUrl: https://github.com/terasky-oss
+category: Kubernetes
+description: View Rightsizing and resource usage data for your Kubernetes workloads right from within Backstage!
+documentation: https://github.com/TeraSky-OSS/backstage-plugins/tree/main/plugins/scaleops-frontend
+iconUrl: https://parsers.vc/logo/71a9c12c-230f-4f96-8d2a-70c637b024ef-3.png
```

Comment:
Do you have permission to use this logo?

Reply:
updated the icon URI to a public github image

Reply:
Just because it's public doesn't mean we can use it any way we like, that's been my experience. Can you share some documentation from them allowing this use, please?

Reply:
i will talk to them tomorrow and get approval from them. i will then just add the image to my plugin repo and link from there or give the URL they prefer.

Reply:
Ive discussed with them and have approval from them to use the logo